tslint.json

```diff
@@ -0,0 +1,113 @@
+{
```

Comment:
I recommend removing this config and using the opinionated linter [prettier ](https://www.npmjs.com/package/prettier) instead. As you prefer,